api-specificatie/openapi.yaml

```diff
@@ -1089,15 +1097,22 @@ components:
           $ref: '#/components/schemas/NummeraanduidingHalBasis'
         woonplaats:
           $ref: '#/components/schemas/WoonplaatsHalBasis'
+    AdresIdentificatiesArray:
+      type: object
+      properties:
+        adresIdentificatie:
+          type: string
+        indicatieNevenadres:
+          type: boolean
     AdresseerbaarObject:
       type: object
       description: Een adresseerbaarobject is een standplaats, ligplaats of verblijfsobject.
       properties:
-        adresseerbaarObjectIdentificatie:
+        identificatie:
           description: unieke identificatie van het adresseerbaar object
           type: string
           example: "01230167890123456"
-        adresseerbaarObjectType:
+        type:
```

Comment:
wanneer we afwijken van de naam in het informatiemodel/gegevenswoordenboek, is het m.i. een goede gewoonte de naam uit het informatiemodel als title op te nemen.

Reply:
toegevoegd